src/libopensc/pkcs15-itacns.c

```diff
@@ -30,17 +30,25 @@
 #include <config.h>
 #endif
 
-#include "pkcs15.h"
-#include "log.h"
-#include "cards.h"
-#include "itacns.h"
 #include <stdlib.h>
 #include <string.h>
 #include <stdio.h>
+
+#ifdef ENABLE_OPENSSL
+#include <openssl/x509.h>
+#include <openssl/x509v3.h>
+#endif
+
+#include "libopensc/internal.h"
+#include "libopensc/pkcs15.h"
+#include "libopensc/log.h"
+#include "libopensc/cards.h"
+#include "libopensc/itacns.h"
```

Comment:
In master All  card-\<card\>.c (38 files)  and pkcs15-\<card\>.c (26 files) used #include "internal.h"
pkcs15-iatcns.c was the only pkcs15-\<card\>.c  that did NOT include internal.h. 

I will change  card-gpk.c, pkcs15-prkey.c, pkcs15-pubkey.c and pkcs15-itacns.c back to use 
#include "internal.h"
rather then 
#include "libopensc/internal.h"
And remove all the other "libopensc/" in these files when I moved the includes.  

Note that in master, there is many other inconsistencies on how includes are done for example try this:
cd src/libopensc
grep "include.*log.h" *
A subject for another PR to clean this up.